packages/react-components/src/components/ChatMessage/MyMessageComponents/ChatMyMessageComponentAsMessageBubble.tsx

```diff
@@ -261,7 +265,10 @@ const MessageBubble = (props: ChatMyMessageComponentAsMessageBubbleProps): JSX.E
           data-ui-id="chat-composite-message"
           author={<Text className={chatMessageDateStyle}>{message.senderDisplayName}</Text>}
```

Comment:
Did we wanted to update `chatMessageDateStyle` to `chatMessageAuthorStyle`

Reply:
Do you have more context/ticket for this update?

Reply:
Oh, I see. `chatMessageAuthorStyle ` is only being updated for prettier fix. Nothing change for the author style in my PR. I'm also not sure why `chatMessageDateStyle` is applied to `auther` 😕